0009-salt-working-groups.md

```diff
@@ -0,0 +1,125 @@
+- Feature Name: Working Group
+- Start Date: 2019-05-16
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Engage with Salt community to get feedback on working group 
+
+# Motivation
+[motivation]: #motivation
+
+Salt's first working groups will convene by mid of June 2019. In order to benefit the participants of current and future working group as well as those administering them, all the details of working group will be added as a README document in saltstack/salt/community/working-group folder. 
+
+
+# Design
```

Comment:
Design section is empty. Remove it?

Reply:
The entire section under the Working Groups belongs to Design section.

Reply:
Then you'll want to prefix the subheadings with `##`.

If you're on a Mac, I highly recommend [MacDown](https://macdown.uranusjr.com/) for editing Markdown files.

Reply:
If everything beneath `# Design` belongs to that header, then all of the sub(-sub...) headings will need one more `#`

Reply:
Went with your earlier suggestion of removing #design heading